plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -154,9 +155,11 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
                     .setSeconds( task.config.getTime().toSeconds() )
             )
 
-        final disk = task.config.getDisk() ?: executor.config.bootDiskSize
-        if( disk )
-            computeResource.setBootDiskMib( disk.getMega() )
+        def disk = task.config.getDiskResource()
+        if( disk && !disk.type )
```

Comment:
Worth to comment on the reason of `!disk.type`, otherwise it's quite cryptic this code

Reply:
Done